.ruby-version

```diff
@@ -0,0 +1 @@
+3.2.1
```

Comment:
Do we need this file?

Reply:
Ruby is needed for KMM development and its KDoctor tool. I had to add Ruby for KDoctor pass.

https://itnext.io/getting-started-with-kotlin-multiplatform-mobile-f047f8973df

Reply:
@alexander-yevsyukov I would then do the following:

* add this point to `README.md` — so that developers install Ruby by themselves, especially on Windows; 
* adjust the GitHub Actions workflows to install desired Ruby version explicitly;
* add this particular file to `.gitignore` in favour of local development environment specific to each dev workstation.

Let's discuss vocally.

Reply:
I've deleted the file. Let's return to this matter when `logging` is extracted and turns true KMM.